frontend/src/metabase/components/LoadingAndGenericErrorWrapper.jsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+import { GenericError } from "metabase/containers/ErrorPages";
+
+import LoadingAndErrorWrapper from "./LoadingAndErrorWrapper";
+
+export default function LoadingAndGenericErrorWrapper(props) {
+  return (
+    <LoadingAndErrorWrapper
+      {...props}
+      renderError={details => <GenericError details={details} />}
+    />
+  );
```

Comment:
Thank @kulyk for [the suggestion on Slack](https://metaboat.slack.com/archives/C505ZNNH4/p1649776410749209?thread_ts=1649773985.732709&cid=C505ZNNH4).